tests/contrib/flask/test_flask_snapshot.py

```diff
@@ -33,12 +39,11 @@ def flask_wsgi_application():
 @pytest.fixture
 def flask_command(flask_wsgi_application, flask_port):
     # type: (str, str) -> List[str]
-    cmd = "ddtrace-run flask run -h 0.0.0.0 -p %s" % (flask_port,)
+    cmd = "ddtrace-run flask run -h 0.0.0.0 -p %s" % flask_port
     return cmd.split()
 
 
-@pytest.fixture
-def flask_env(flask_wsgi_application):
+def flask_default_env(flask_wsgi_application):
```

Comment:
Since now the snapshot test will need two different envs depending on if they're ASM or not I turned the previous `flask_env` fixture into two non-fixture functions (didn't see a simple and elegant way to parametrize a fixture directly) and use it as an argument to `@parametrize` on the tests. A nice uninstended side-effect is that now the snapshot names contain the environment used.